.travis.yml

```diff
@@ -51,6 +51,7 @@ install:
           $HOME/.cabal/packages/hackage.haskell.org/00-index.tar;
    fi
  - travis_retry cabal update -v
+ - cabal install cpphs
```

Comment:
It's for the old eq specialize rules, i have updated to use @andrewthad  's solution now, so this's not needed.